controllers/package_controller.go

```diff
@@ -124,8 +144,19 @@ func (r *PackageReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 			return ctrl.Result{RequeueAfter: retryLong}, err
 		}
 
-		managerContext.Source, managerContext.Version, err = bundle.FindSource(managerContext.Package.Spec.PackageName, managerContext.Package.Spec.PackageVersion)
+		targetVersion := managerContext.Package.Spec.PackageVersion
+		if targetVersion == "" {
+			targetVersion = api.Latest
+		}
+		pkgName := managerContext.Package.Spec.PackageName
+		managerContext.Source, err = bundle.FindSource(pkgName, targetVersion)
+		managerContext.Package.Status.TargetVersion = printableTargetVersion(managerContext.Source, targetVersion)
 		if err != nil {
+			//TODO add a link to documentation on how to make a bundle active.
+			managerContext.Package.Status.Detail = fmt.Sprintf("Package %s@%s is not in the current active bundle. Did you forget to activate the new bundle?", pkgName, targetVersion)
```

Comment:
```suggestion
			managerContext.Package.Status.Detail = fmt.Sprintf("Package %s@%s is not in the active bundle (%s). Did you forget to activate the new bundle?", bundle.Version, pkgName, targetVersion)
```

I wouldn't output the second sentence unless I knew there was a newer bundle on the system. I feel like error messages that are misleading are worse than terse ones.